Sources/FaunaDB/Query.swift

```diff
@@ -26,6 +26,104 @@ public struct Ref: Fn {
 
 // MARK: Basic Forms
 
+/// Returns a native reference to classes. This allows for example,
+/// paginate over all classes in a database.
+///
+/// [Reference](https://fauna.com/documentation/queries#basic_forms).
+public struct Classes: Fn {
+
+    var call: Fn.Call
+
+    /// - Parameter scope: the scope database or nothing for the current database.
+    public init(_ scope: Expr = NullV()) {
```

Comment:
I think we should not have `NullV` as a default parameter at the surface area of our drivers. That could make people wonder what are those values for when we actually want them to be more focused on the expression level constructors.